airflow/sentry.py

```diff
@@ -178,8 +179,12 @@ def wrapper(_self, *args, **kwargs):
 
                 with sentry_sdk.push_scope():
                     try:
-                        # Is a LocalTaskJob get the task instance
-                        if hasattr(_self, "task_instance"):
+                        ti_from_kwargs = kwargs.get("ti", None)
```

Comment:
whyntask instance is mandatory for sentry error logging?

Reply:
It's needed by `add_tagging` and `add_breadcrumbs`